internal/config/config.go

```diff
@@ -0,0 +1,26 @@
+// Package config works with application configurations.
+package config
+
+import "os"
+
+// Config represents the application configurations.
+type Config struct {
+	Username   string
+	Password   string
+	DBName     string
+	Host       string
+	Port       string
+	SSLMode    string
+	PrivateKey string
+}
+
+// Load loads the necessary configurations from the .env file.
+func (c *Config) Load() {
```

Comment:
Looks like it doesn't actually load the config form `.env` file but rather gets it from the environment variables. It's fine, but I guess it would be helpful for local development to load local config from `.env` or some `yaml` file.

Reply:
Take a look at this lib https://github.com/kelseyhightower/envconfig